core/src/main/scala/kafka/server/DynamicBrokerConfig.scala

```diff
@@ -535,24 +539,24 @@ class DynamicBrokerConfig(private val kafkaConfig: KafkaConfig) extends Logging
     if (changeMap.nonEmpty || deletedKeySet.nonEmpty) {
       try {
         val customConfigs = new util.HashMap[String, Object](newConfig.originalsFromThisConfig) // non-Kafka configs
-        newConfig.valuesFromThisConfig.keySet.forEach(customConfigs.remove(_))
-        reconfigurables.foreach {
+        newConfig.valuesFromThisConfig.keySet.forEach(k => customConfigs.remove(k))
+        reconfigurables.forEach( {
```

Comment:
nit: You can remove the parenthesis. `forEach { ...` should work.

Reply:
Cheers :)